src/components/ProjectCard.tsx

```diff
@@ -0,0 +1,157 @@
+// Todo: re-integrate Personnel
+
+import React from 'react';
+import {
+  Card,
+  CardHeader,
+  // CardMedia,
+  CardContent,
+  CardActions,
+  Typography,
+  Collapse,
+  List,
+  ListItem,
+  ListItemText,
+  IconButton,
+  styled,
+  createStyles,
+  makeStyles,
+  Theme,
+} from '@material-ui/core';
+
+import { Link } from 'react-router-dom';
+
+import clsx from 'clsx';
+
+import { Person } from 'types/person';
+import { Task } from 'types/task';
+import { Project } from 'types/project';
+
+import { ExpandMore as ExpandMoreIcon } from '@material-ui/icons';
+
+interface ProjectCardProps {
+  project: Project;
+}
+
+const useStyles = makeStyles((theme: Theme) =>
+  createStyles({
+    expand: {
+      transform: 'rotate(0deg)',
+      marginLeft: 'auto',
+      transition: theme.transitions.create('transform', {
+        duration: theme.transitions.duration.shortest,
+      }),
+    },
+    expandOpen: {
+      transform: 'rotate(180deg)',
+    },
+  })
+);
+
+const CustomCard = styled(Card)(() => {
+  return {
+    maxWidth: '350px',
+    margin: '1rem',
+  };
+});
+
+const ProjectCard = (props: ProjectCardProps): JSX.Element => {
+  const { project } = props;
+
+  const { tasks } = project; // Todo: destructure here
+
+  const [expanded, setExpanded] = React.useState(false);
+
+  const classes = useStyles();
+
+  const handleExpandClick = () => {
+    setExpanded(!expanded);
+  };
+
+  const displayTasks = (tasks: Task[]): JSX.Element => {
+    return (
+      <List>
+        {tasks.map((task, index) => {
+          return (
+            <ListItem key={index}>
+              <Link to={`/tasks/${task.id}`}>
+                <ListItemText>
+                  {task.name}:
+                  {task.description.length > 40
+                    ? task.description.slice(0, 40)
+                    : task.description}
+                </ListItemText>
+              </Link>
+            </ListItem>
+          );
+        })}
+      </List>
+    );
+  };
+
+  const displayList = (strArr: string[]): JSX.Element => {
+    return (
+      <List>
+        {strArr.map((str, index) => {
+          return (
+            <ListItem key={index}>
+              <ListItemText>{str}</ListItemText>
+            </ListItem>
+          );
+        })}
+      </List>
+    );
+  };
+
+  return (
+    <CustomCard>
+      <Link to={`/projects/${project.id}`}>
+        <CardHeader
+          title={
+            project.title.length > 15
+              ? project.title.slice(0, 15)
+              : project.title
+          }
+          subheader={project.deadline}
+        />
+      </Link>
+      <CardContent>
+        <Typography>{project.description}</Typography>
+      </CardContent>
+      <CardActions>
+        <IconButton
+          className={clsx(classes.expand, {
+            [classes.expandOpen]: expanded,
+          })}
+          onClick={handleExpandClick}
+          aria-expanded={expanded}
+          aria-label="show more"
```

Comment:
aria items are used as a last resort for screen readers to know what to say when something is shown.  Is that why they are placed here?